elm/Main.elm

```diff
@@ -258,7 +262,7 @@ boundedPager radius f p =
 
 viewCallLink : Int -> String -> Html Msg
 viewCallLink index rendered =
-    a [ href "javascript:()", onClick (SelectCall index) ] [ text rendered ]
+    a [ href ("#call/" ++ toString index) ] [ text rendered ]
```

Comment:
Do we need any fancy escaping here? Does it still work if you have `/` as the name or similar?

Reply:
Not sure, but I would assume that Elm takes care of escaping the `rendered` text child. Or are you thinking of the href link ?

Reply:
Yes, the href link, which might need different escaping with URL encoding for certain characters (or might not - happy to wait til we see the problem).

Reply:
Well, `index` is an int, so provided the type system doesn't lie, I think no injection is possible here right?

Reply:
Oh, I hadn't noticed that - makes sense - an Int guarantees it is safe.